### PR TITLE
Milestone: Query and update animals table

### PR DESCRIPTION
**Pull request by Kashif-Saleem-Ghuman**
In this milestone 💪 

- [ ] Add new species column
- [ ] Add required data
- [ ] Make required queries
- [ ] Screenshots:

animals_never_tried_to_escape
avg_escape_attempts_as_per_birth_date_between_1990_2000
avg_weight
delete
delete_rollback
digimon_pokemon
min_max_weight_type_animal
most_escapes_nuetered_nonNuetered
no_of_animals
rollback
unspecified

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
+
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg)
+values ('Plantmon', '2021-11-15', 2, True, 5.7);
+
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg)
+values ('Squirtle', '1993-04-02', 3, False, 12.13);
+
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg)
+values ('Angemon', '2005-06-12', 1, True, 45);
+
```
Comment (hiromataba) [inline data.sql:27]: - The weight of "Angemon" is `-45kg` not 45kg

image


- The weight of "Squirrel" is `-12.13kg` not 12.13kg

image

File: data.sql [hunk 2]
```diff
@@ -1,14 +1,35 @@
...
+values ('Pikachu', '2021-01-07', 1, false, 15.0) 
 
-insert into animals (id, name, date_of_birth, escape_attempts ,  neutered, weight_kg )
-values (4,  'Devimon', '2017-06-12', 5, true, 11)
\ No newline at end of file
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg )
+values ('Devimon', '2017-06-12', 5, true, 11)
+
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg)
+values ('Charmander', '2020-02-08', 0, False, 11);
```
Comment (hiromataba) [inline data.sql:17]: - [ ] The weight of "Charmander" is -11kg not 11kg. 

image

File: data.sql [hunk 3]
```diff
@@ -1,14 +1,35 @@
...
+
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg)
+values ('Plantmon', '2021-11-15', 2, True, 5.7);
```
Comment (hiromataba) [inline data.sql:20]: - [ ] The weight of "Plantmon" is -5.7kg not 5.7kg.

image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @Kashif-Saleem-Ghuman   :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql [hunk 4]
```diff
@@ -8,3 +8,86 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg>10.5
...
+SELECT species, MIN(weight_kg) as min_weight, MAX(weight_kg) as max_weight
+FROM animals
+GROUP BY species;
+
+SELECT species, AVG(escape_attempts) AS avg_escape_attempts
+FROM animals
+WHERE date_of_birth BETWEEN DATE '1990-01-01' AND DATE '2000-12-31'
+GROUP BY species;
+
+-- correct wightt of Angemon
```
Comment (1GORDON) [inline queries.sql]: Great job @Kashif-Saleem-Ghuman 🤝

- [ ] [Optional] As stated by the previous reviewer, kindly add update the weight_kg value of `Squirtle` and add these changes to the data.sql file.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Kashif-Saleem-Ghuman ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**